### PR TITLE
【KernelGen】 delete rmsnorm from experimental_ops

### PR DESCRIPTION
**Pull request by factnn**
- rmsnorm Triton implementation underperforms PyTorch official API
- Performance test shows only 1/15 cases faster, 14/15 cases slower (0.28-0.38x speedup)
- PyTorch torch.nn.functional.rms_norm provides better performance
- Removed files: rmsnorm.py, rmsnorm_test.py
- Updated __init__.py to remove rmsnorm import
- Updated README.md to remove rmsnorm examples

### CODE REVIEW

File: src/flag_gems/experimental_ops/README.md [hunk 1]
```diff
@@ -20,12 +20,10 @@ with flag_gems.use_gems():
 ```
 src/flag_gems/experimental_ops/
 ├── __init__.py                 # Module initialization
-├── rmsnorm.py          # Example operator implementation
-├── [other_operators].py   # Additional operators
+├── [your_operator].py   # Operator implementations
 ├── exp_tests/                 # Accuracy test and performance test
```
Comment (tengqm) [inline src/flag_gems/experimental_ops/README.md]: this should be `tests/experimental_ops/`

**Review by tengqm — APPROVED** [review]
